Tests/Functional/Doctrine/Phpcr/SitemapDocumentProviderTest.php

```diff
@@ -48,6 +48,7 @@ public function testDocumentOrder()
         $this->assertEquals(
             array(
                 '/test/content/sitemap-aware',
+                1 => '/test/content/sitemap-aware-last-mod-date',
```

Comment:
Stupid copy from Test expectation. Whant to fix it and my power goes down just before boarding. Will do the fixes later.